docs/guides/integration/github.md

```diff
@@ -137,14 +137,55 @@ steps:
 
 ## Using `uv pip`
 
-If using the `uv pip` interface instead of the uv project interface, uv requires a virtual environment by default. To allow installing packages into the system environment, use the `--system` flag on all `uv` invocations or set the `UV_SYSTEM_PYTHON` variable, e.g.:
+If using the `uv pip` interface instead of the uv project interface, uv requires a virtual environment by default. To allow installing packages into the system environment, use the `--system` flag on all `uv` invocations or set the `UV_SYSTEM_PYTHON` variable.
+
+### Setting `UV_SYSTEM_PYTHON`
+
+`UV_SYSTEM_PYTHON` variable can be defined in various scopes:
+
+i. Workflow-wide Environment Variables
+
+Set the variable for the entire workflow by defining it at the top level:
+
+```yaml title="example.yml"
+env:
+  UV_SYSTEM_PYTHON: 1
+
+jobs: ...
+```
+
+ii. Job-specific Environment Variables
+
+Set the variable for a specific job within the workflow:
+
+```yaml title="example.yml"
+jobs:
+  install_job:
+    env:
+      UV_SYSTEM_PYTHON: 1
+    ...
+```
+
+Or using a shell command:
```

Comment:
I think this shell command might not work on Windows, should we mention it's unix specific?

Reply:
Hm interesting yeah seems like it might need `>> $env:GITHUB_ENV` on Windows. Let's just drop it — no reason to do it this way really.

Reply:
Done